carma/launch/environment.launch

```diff
@@ -45,8 +45,8 @@
   <!-- Object Tracking and World Model -->
 
   <!-- CARMA Motion Computation Node -->
-  <group>
-    <include file = "$(find motion_computation)/launch/external_object_predictions.launch"/>
+  <group unless="$(arg simulation_mode)">
```

Comment:
Remove conditional. The simulation publishes to this node so it should always be running